libraries/AP_Notify/Display.cpp

```diff
@@ -377,6 +377,17 @@ void Display::update()
     }
     timer = 0;
 
+    // If we have received an override from scripting recently allow update
+    if (AP_HAL::millis() - _last_scr_override <= _script_timeout_ms) {
```

Comment:
Won't this block the display for the first 5 seconds after boot? You probably need a boolean for "we have overrides" and then once outside the time window set the boolean back to false.